docs/ldml/tr35-keyboards.md

```diff
@@ -287,19 +301,22 @@ Attribute values escaped in this manner are annotated with the `<!--@ALLOWS_UESC
 
 ## File and Directory Structure
 
-* New collection of layouts that are prescriptive, and define the common core for a keyboard that can be consumed as data for implementation on different platforms will be included in the CLDR repository. This collection will be in a different location than the existing CLDR keyboard files under main/keyboards. We should remove the existing data files, but keep the old DTD in the same place for compatibility, and also so that conversion tools can use it to read older files.
-* New layouts will have version metadata to indicate their specification compliance versi​​on number. For this tech preview, the value used must be `techpreview`.
+* In the future, new layouts will be included in the CLDR repository, as a way for new layouts to be distributed in a cross-platorm manner. The process for this repository of layouts has not yet been defined, see [CLDR-17253](https://unicode-org.atlassian.net/browse/CLDR-17253) for up-to-date information.  Layouts which are currently in the repository are intended as sample data only.
+
```

Comment:
I could expect to see just sample data in the repository when the spec is in tech preview. Once it is approved, however, I would expect to see real data there (and no sample data).

Reply:
The challenge is that we haven't defined the process for inclusion in the common repository yet. 

The Bengali and Maltese keyboards are real keyboards, not toys. The French azerty keyboard is a test, that could be withdrawn for now.

Perhaps something else here, saying that the process for inclusion is still being defined, and strike the section about sample data?

Reply:
I've updated this to remove the mention of sample data. I'll plan to remove the only keyboard that's only sample data, the azerty one.

Reply:
Mark, I've re-added the DTDs so that sample files pass.  CLDR tooling breaks without a DTD.  Split this out to ticket https://unicode-org.atlassian.net/browse/CLDR-17371